### DISCUSSION TITLE
How to run `@agoric/orchestration` examples?

### DISCUSSION THREAD

**Discussion by anilhelvaci** [Agoric/agoric-sdk#9398] [Q&A]
Currently,  there are a few examples listed in [src/examples](https://github.com/Agoric/agoric-sdk/tree/master/packages/orchestration/src/examples). I'm wondering how's the configuration for these examples look like?

* https://github.com/Agoric/agoric-sdk/issues/8896 talks about some docker image that has the universe we need, what's the current state on that? Do we have something to play around?

* If no a3p code for `TEST` phase present, how do the interaction with one those examples planned/carried out?

**Comment by turadg** [comment]
Those do not yet work. They are examples of how it will work.

https://github.com/Agoric/agoric-sdk/tree/master/packages/orchestration/test/examples has passing unit tests, which fake many of the lower level capabilities.

The current state of #8896 is as the issue describes. Open and with no accompanying PR. The comments describe research to make it work.

There may be no a3p tests for these examples because they're not meant to be deployed on chain. There will be other integration tests and A3P will have tests of the capabilities to be deployed on chain (through core evals in an upgrade). These examples or others may be used in the a3p test image of those proposals.
